src/components/InitialURLContextProvider.tsx

```diff
@@ -15,16 +14,7 @@ type InitialURLContextProviderProps = {
 };
 
 function InitialURLContextProvider({children, url}: InitialURLContextProviderProps) {
-    const [initialURL, setInitialURL] = useState(url);
-    useEffect(() => {
-        if (initialURL) {
-            return;
-        }
-        Linking.getInitialURL().then((initURL) => {
-            setInitialURL(initURL as Route);
-        });
-    }, [initialURL]);
-    return <InitialURLContext.Provider value={initialURL}>{children}</InitialURLContext.Provider>;
```

Comment:
How come we removed this in this PR? This ended up causing https://github.com/Expensify/App/issues/41132